### PR TITLE
Added 'quiet' docker arg support

### PR DESCRIPTION
**Pull request by jj-mino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,6 +210,7 @@ Following inputs can be used as `step.with` keys
 | `ssh`               | List     | List of SSH agent socket or keys to expose to the build |
 | `tags`              | List/CSV | List of tags |
 | `target`            | String   | Sets the target stage to build |
+| `quiet`             | String   | Suppress the build output and print image ID on success |
```
Comment (fearphage) [inline README.md:213]: Looks like the previous options were in alphabetical order.

File: src/context.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ export interface Inputs {
   noCache: boolean;
   outputs: string[];
   platforms: string[];
+  quiet: boolean;
```
Comment (fearphage) [inline src/context.ts:29]: Alphabetize here too.

File: src/context.ts [hunk 3]
```diff
@@ -74,6 +75,7 @@ export async function getInputs(defaultContext: string): Promise<Inputs> {
     load: core.getBooleanInput('load'),
     network: core.getInput('network'),
     noCache: core.getBooleanInput('no-cache'),
+    quiet: core.getBooleanInput('quiet'),
```
Comment (fearphage) [inline src/context.ts:78]: Alphabetize here too.

**Review by fearphage — CHANGES_REQUESTED** [review]
You should try to maintain the current state of things.

**Comment by crazy-max** [comment]
As discussed in #501, we are not going to add the `quiet` flag as it does not make much sense for the action itself.

**Pull request closed without merging**